LDAP-Auth/Api/Models/LdapUser.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace Jellyfin.Plugin.LDAP_Auth.Api.Models
+{
+    /// <summary>
+    /// LdapUser class.
+    /// </summary>
+    public class LdapUser
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LdapUser"/> class.
+        /// </summary>
+        public LdapUser()
+        {
+            LinkedJfUserId = Guid.Empty;
+            LdapUid = string.Empty;
+        }
+
+        /// <summary>
+        /// Gets or sets the linked Jellyfin user id.
+        /// </summary>
+        public Guid LinkedJfUserId { get; set; }
```

Comment:
I would prefer to have the variable be named `LinkedJellyfinUserId`

Reply:
Renamed